examples/mutators/set-annotation/advanced/README.md

```diff
@@ -1,22 +1,17 @@
-# Set Annotation Advanced Example
+# set-annotation: Advanced Example
 
 The `set-annotation` function adds annotations to KRM resources.
 
-In this example, we use `SetAnnotationConfig` to configure the function. The
-desired annotations are provided using the `annotations` field.
-
-We also specify `fieldSpecs` for our CRD with group as `dev.example.com`,
-version as `v1` and kind as `MyResource`. The annotations will also be added to
-field `.spec.selector.annotations`.
+We use the following `SetAnnotationConfig` to configure the function.
 
 ```yaml
-apiVersion: kpt.dev/v1beta1
+apiVersion: fn.kpt.dev/v1alpha1
```

Comment:
This depends on what we use in https://github.com/GoogleContainerTools/kpt-functions-catalog/pull/181